conans/client/generators/ycm.py

```diff
@@ -164,23 +172,25 @@ def FlagsForFile( filename, **kwargs ):
 
     @property
     def filename(self):
-        return '.ycm_extra_conf.py'
+        pass
 
     @property
     def content(self):
         def prefixed(prefix, values):
             return [prefix + x for x in values]
 
-        flags = ['-x', 'c++']
-        flags.extend(self.deps_build_info.cppflags)
-        flags.extend(prefixed("-D", self.deps_build_info.defines))
-        flags.extend(prefixed("-I", self.deps_build_info.include_paths))
+        conan_flags = {
```

Comment:
why change -I to -system?

Reply:
I think the idea is to avoid warnings coming from third-party libraries.

Reply:
exactly.